### DISCUSSION TITLE
Power Limits Factory default button.

### DISCUSSION THREAD

**Discussion by paulusgi** [seerge/g-helper#206] [Ideas]
It would be nice to have a button to change the power limits to default values within each power profile. Because the Factory Default button only affects fan curves right now.
If there is a way to read the BIOS defaults, I think it's a not-so-complicated feature to add and would be great in my opinion.

Thanks a lot for that amazing software. ^_^

**Comment by seerge** [comment]
Hello, "factory defaults" resets everything, including PPTs (i.e. it just doesn't apply PPTs or fan curves to a default profile)
There is no way to read default PPTs from bios anyhow, so sliders won't change, but checkbox under them - does

**Reply by paulusgi** [reply]
Thanks for fast reply. Then the values change in the background? So there is no way to be able to read the defaults somewhere to change the sliders values too?

**Reply by seerge** [reply]
If "apply power limits" is unchecked - it will uses default limits (whatever they are for each mode / specific model / bios). If checked - it will set limits from current sliders.

There is no way to "read" ppts :) (even reading fan curve is troublesome and is not supported by hardware on some models like g15, but does on g14 ...)

**Reply by paulusgi** [reply]
ok now I understand perfectly. So, the only way (to make the sliders change) would be to make a database of each model with the factory default ppt values and I guess it's not worth it. x)

Thanks again.
